test/module/irohad/consensus/yac/mock_yac_hash_provider.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_HASH_PROVIDER_HPP
```

Comment:
```suggestion
#ifndef IROHA_MOCK_YAC_HASH_PROVIDER_HPP
```
since class name is the following
https://github.com/hyperledger/iroha/blob/615290a35e2bb93b71630a04a6c9f5817af707f0/test/module/irohad/consensus/yac/mock_yac_hash_provider.hpp#L17